docs/Secure-Coding-Guide-for-Python/CWE-693/CWE-798/noncompliant01.py

```diff
@@ -1,5 +1,40 @@
 # SPDX-FileCopyrightText: OpenSSF project contributors
 # SPDX-License-Identifier: MIT
-""" Non-compliant Code Example """
-databaseIPAddress = "192.168.0.1"
-print(databaseIPAddress)
+"""Non-compliant Code Example"""
+
+import logging
+import unittest
+
+logging.basicConfig(encoding="utf-8", level=logging.DEBUG)
+
+
+def front_end():
+    """Dummy method demonstrating noncompliant implementation"""
+    # A noncompliant implementation would typically hardcode server_config
+    # and load it from a project global python file or variable
+    server_config = {}
+    server_config["IP"] = "192.168.0.1"
+    server_config["PORT"] = "192.168.0.1"
```

Comment:
Value is not a port:
```suggestion
    server_config["PORT"] = "8080"
```